.github/workflows/dart.yml

```diff
@@ -82,7 +82,25 @@ jobs:
       - uses: ./
         with:
           sdk: ${{ matrix.sdk }}
-          architecture: ia32
+
+      - name: Run hello world
+        run: |
+          echo "main() { print('hello world'); }" > hello.dart
+          dart hello.dart
+      - run: dart --version
+
+  # Test using wildcard versions for the sdk parameter.
+  test_release_wildcards:
```

Comment:
Nit: `test_latest_patch_release` (and `# Test getting the latest patch release for a major.minor sdk parameter.`)